orca-web/config/orca.yml

```diff
@@ -32,6 +32,10 @@ kayenta:
   enabled: ${services.kayenta.enabled:false}
   baseUrl: ${services.kayenta.baseUrl:http://localhost:8090}
 
+services:
+  fiat:
+    baseUrl: http://localhost:7003
```

Comment:
Thanks for this!

I'd suggest adding `services.fiat.enabled: false` as the default too, to ensure Orca doesn't fail to start if Fiat isn't also running locally.